keyboards/hackpad/info.json

```diff
@@ -0,0 +1,78 @@
+{
+    "manufacturer": "Nico Stuhlmueller",
+    "keyboard_name": "hackpad",
+    "maintainer": "ThePurox",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgblight": true,
+        "encoder" : true
+    },
+    "matrix_pins": {
+        "cols": ["B1", "B3", "B2", "B6"],
+        "rows": ["C6", "D7", "E6", "B4", "B5"]
+    },
+    "encoder" : {
+        "rotary" : [
+            {"pin_a" : "F5", "pin_b" : "F4"}
+        ]
+    },
+    "url": "https://git.imaginaerraum.de/Purox/Hackpad",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "ws2812": {
+        "pin": "D4"
+    },
+    "rgblight" : {
+        "led_count" : 16,
+        "hue_steps" : 20,
+        "brightness_steps" : 10,
+        "saturation_steps" : 20,
```

Comment:
These values differ so little from the defaults that they dont add a ton of value
```suggestion
```

Reply:
done